### ISSUE TITLE
No 11.6.1 from Catalina?

### ISSUE THREAD

**Issue by telepathine-ltd** [grahampugh/erase-install#160]
Hi all, thanks for taking the time to help here. We're deploying the latest version (24) of this script in our environment; we're testing using it to upgrade everyone to 11.6.1 remotely. However... on all 3 MacBooks we've run this on so far, none of them seem to be showing 11.6.1 as a valid upgrade from 10.15.7 Catalina. Checking the Jamf log shows that the script doesn't see 11.6.1, just 11.6 and Monterey. Is this expected behavior?

**Comment by grahampugh** [comment]
What happens when you run `erase-install.sh --list` from Catalina? Do you see 11.6.1? I suspect Apple never added 11.6.1 to the Catalina catalog.

**Comment by linux2229** [comment]
Hi Graham, I'm also seeing the same behavior that telepathine-ltd  described, on intel machines running 10.15.7 or Big Sur  , I dont see 11.6.1, but if I use a M1 machine with Big Sur , the script is able to get 11.6.1. I'm also using the latest script to upgrade 10.15.7 machines to Big Sur but only getting 11.6 or Monterey 
When run erase-install --list from an intel Mac running 10.15.7 , I get this:

![erase-install](https://user-images.githubusercontent.com/10354347/142038402-cdce3b33-e142-48b7-b81b-7e1e4cf95bf8.png)

any ideas?

**Comment by grahampugh** [comment]
Yep. You can see from the list that Apple has not provided 11.6.1 in the catalogs. What happens if you run `erase-install.sh --list-full-installers`? (this is the same as running `softwareupdate --list-full-installers`) Do you see 11.6.1? I guess not?

So you either have to accept 11.6 because that it what Apple are providing, or you have to override the catalog and hope it works. For example, you could try adding:

```
--catalogurl=https://swscan.apple.com/content/catalogs/others/index-10.16-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog
```

**Comment by linux2229** [comment]
I ran the script with the --catalog parameter as you suggested, and I do see the 11.6.1 listed but the script chose the Monterey installer option:
![erase-install2](https://user-images.githubusercontent.com/10354347/142046509-13b988e0-b6a4-4523-b64e-f771af899ce2.png)

**Comment by grahampugh** [comment]
Yes of course, if you don't specify a version or an os, you get the latest valid version.
